README.md

```diff
@@ -1,70 +1,82 @@
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Live
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[LiveDemo](https://ridaarif98.github.io/metrics-webapp-capstone/)
 
-### `npm run eject`
+### Setup
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+```cmd
+ git clone https://github.com/ridaarif98/metrics-webapp-capstone.git
+ cd metrics-webapp-capstone
+```
+### Install
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+```cmd
+npm install
+npm run build
+```
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+### Usage
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+```cmd
+npm start
+```
 
-## Learn More
+### To run Test
+```cmd
+npm run test
+```
+## Author
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+👤 **Rida Arif**
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- GitHub: [@ridaarif98](https://github.com/ridaarif98)
+- Twitter: [@ridaarif](https://twitter.com/Rida29984906)
+- LinkedIn: [Rida Arif](https://www.linkedin.com/in/rida-arif-90945520b/)
 
-### Code Splitting
+## 🤝 Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Contributions, issues, and feature requests are welcome!
 
-### Analyzing the Bundle Size
+Feel free to check the [issues page](https://github.com/ridaarif98/metrics-webapp-capstone/issues).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## Show your support
 
-### Making a Progressive Web App
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## Acknowledgments
 
-### Advanced Configuration
+- Microverse for giving me this chance
+- To my Morning Team and coding partners who kept my morale up!
+- The amazing code reviewers for making me improve every day :thumbsup:
```

Comment:
- [ ] [Optional] I can see you have worked hard on your Readme but there are Grammar mistakes. please work on those.